### ISSUE TITLE
More descriptive alt attribute control

### ISSUE THREAD

**Issue by iseulde** [WordPress/gutenberg#1509]
Suggestion:

<img width="304" alt="screenshot 2017-06-27 18 27 51" src="https://user-images.githubusercontent.com/4710635/27598685-8a5efb70-5b66-11e7-8671-b245b72ded90.png">

Cc @joen for the patterns and @afercia for the content.

**Comment by iseulde** [comment]
As it turns out there's quite a big process to deciding whether or not one should be used and what it should be: https://www.w3.org/WAI/tutorials/images/decision-tree/ (hat tip: @afercia). The task is to bring this over in just a line or two. :)

**Comment by joedolson** [comment]
I think that's pretty good. I'm wondering if it can be made more concise, e.g.

"Describe the purpose of the image. Leave empty if the image is not a key part of the content."

**Comment by nic-bertino** [comment]
It would be fantastic if this was pre-filled with alt text from the media library if it already exists.

I think one of my daily questions is "what should go in my alt?" I think the decision tree link is a great resource, perhaps it is something that can be linked directly from this interface (along with @joedolson's concise copy)?

**Comment by iseulde** [comment]
> It would be fantastic if this was pre-filled with alt text from the media library if it already exists.

It should be already.

I don't think it's bad to link to w3.org. It should be a stable link. :D Alternatively we might want to copy that over to somewhere on WordPress.org.

**Comment by afercia** [comment]
> we might want to copy that over to somewhere on WordPress.org.

Or in the Gutenberg inline help. I guess there should be some inline help somewhere? 🙂

**Comment by afercia** [comment]
See also #1570

**Comment by mtias** [comment]
@jasmussen @karmatosed this seems an easy one to do and close, what do you think?

**Comment by jasmussen** [comment]
I've started work on this one, but I think we should consider adding "help text" as something that can be passed on to the component itself, in this case the TextControl component. So going to do that before I PR.

**Comment by jasmussen** [comment]
Work done on this one. We can close this issue when #3591 gets merged.
